packages/web-components/src/components/va-memorable-date/va-memorable-date.scss

```diff
@@ -13,103 +13,58 @@
 @import '../../mixins/uswds-error-border.scss';
 @import '../../global/formation_overrides';
 @import '../../mixins/focusable.css';
-@import '~@department-of-veterans-affairs/css-library/dist/elements.css';
-
-.usa-memorable-date {
-  margin-top: 16px;
-  // USWDS v3.7.1 set flex-wrap: wrap; This caused the year to always wrap when the month select option was active.
-  flex-wrap: nowrap;
-}
-
-.usa-form-group--month {
-  margin-right: 16px;
-}
-
-.usa-form-group--month-input {
-  width: 48px;
-}
+@import '../../mixins/form-field-error.css';
 
-@media (max-width: 480px) {
-  .usa-form-group--month-select {
-    width: 100%;
-  }
-  .usa-form-group--select {
-    flex: 0 1 50%;
-    width: 50%;
-  }
-}
+@import '~@department-of-veterans-affairs/css-library/dist/elements.css';
 
-:host(:not([uswds='false'])) {
+:host {
+  display: block;
+  font-family: var(--font-source-sans);
```

Comment:
The `va-date` stylesheet was being imported previously for the v1 variation but there was still a few styles that the USWDS version was relying on. We did not need to the entire stylesheet though.

So I have removed that import `@import '../va-date/va-date.css';` and instead added the relevant styles from there like these here.